DAOS-15947 docs: Update admin guide dmg system command syntax (#16261)
daos-stack/daos: docs/admin/pool_operations.md

Reviewer: at line 1286 of this file the sentence
`or target until after the rebuild is complete.`
should probably replace by
`or target until the rebuild is complete.`

Author: I think "after" actually reads better.

### Manual Exclusion

An operator can exclude one or more engines or targets from a specific DAOS pool using the rank(s)
where the target(s) reside, as well as the target index(es) on the rank(s). If a target idx list is
not provided, all selected targets on the engine rank(s) will be excluded.

To exclude multiple targets on a single rank from a pool:

```bash
$ dmg pool exclude --ranks=${rank} --target-idx=${idx1},${idx2},${idx3} <pool_label>
```

To exclude multiple targets on multiple ranks from a pool:

```bash
$ dmg pool exclude --ranks=${rank_range_list} --target-idx=${idx1},${idx2},${idx3} <pool_label>
```

> The pool target reintegrate command accepts 4 parameters:

Reviewer: ```suggestion
The pool target exclude command accepts 4 parameters:
```

> * The label or UUID of the pool that the targets will be reintegrated into.

Reviewer: ```suggestion
* The label or UUID of the pool that the targets will be excluded.
```

    the --force option. Please note that forcing the operation may result in data loss,
    and it is strongly recommended to verify the RF status before proceeding.

#### System Exclude

To exclude ranks or hosts from all pools that they belong to, the 'dmg system exclude'
command can be used. The command takes either a host-set or rank-set:

To exclude a set of hosts from all pools (excludes all ranks on selected hosts):

```Bash
$ dmg system exclude --rank-hosts foo-[001-100]
```

To exclude a set of ranks from all pools:

```Bash
$ dmg system exclude --ranks 1-100
```

Selected ranks will be set to `AdminExcluded` state in system membership (as shown in output of
`dmg system query` command). This will exclude all rank targets from all pools and render the ranks
unusable in pools until `dmg system clear-exclude --ranks=${rank_range_list}` is run to return the
ranks to the previous (normally `Excluded`) state. Then the engine can be restarted to rejoin the

> system and will enter the `Joined` state and ready to be reintegrated into pools.

Reviewer: It could be indicated that the engine will rejoin automatically the system and also indicate how much time it took in general.  Could be helpful to add an example of dmg command usage for checking if the engines have enter the `Joined` state.

Author: time is not easy to determine as it depends on the size of the cluster and lots of other factors. I can add the syntax for system query. Done

Drain operation is not allowed if there are other ongoing rebuild operations, otherwise
it will return -DER_BUSY.

An operator can drain one or more engines or targets from a specific DAOS pool using the rank(s)
where the target(s) reside, as well as the target index(es) on the rank(s). If a target idx list is
not provided, all selected targets on the engine rank(s) will be drained.

To drain multiple targets on a single rank from a pool:

```bash
$ dmg pool drain --ranks=${rank} --target-idx=${idx1},${idx2},${idx3} <pool_label>
```

To drain multiple targets on multiple ranks from a pool:

```bash
$ dmg pool drain --ranks=${rank_range_list} --target-idx=${idx1},${idx2},${idx3} <pool_label>
```

> The pool target reintegrate command accepts 3 parameters:

Reviewer: ```suggestion
The pool target drain command accepts 3 parameters:
```

> * The label or UUID of the pool that the targets will be reintegrated into.

Reviewer: ```suggestion
* The label or UUID of the pool that the targets to be drained.
```
